go.mod

```diff
@@ -1,8 +1,8 @@
 module github.com/cybozu-go/necoperf
 
-go 1.22.0
+go 1.23.0
 
-toolchain go1.22.6
+toolchain go1.23.3
```

Comment:
When I ran `go mod tidy`, it caused some changes to appear. 
Could you please run the `go mod tidy -go=1.23.3` command to update the module?